--- system.conf.orig	2020-06-23 13:09:40.000000000 +0200
+++ system.conf	2020-07-23 12:25:26.909927095 +0200
@@ -38,8 +38,8 @@
 #DefaultTimerAccuracySec=1min
 #DefaultStandardOutput=journal
 #DefaultStandardError=inherit
-#DefaultTimeoutStartSec=90s
-#DefaultTimeoutStopSec=90s
+DefaultTimeoutStartSec=15s
+DefaultTimeoutStopSec=15s
 #DefaultTimeoutAbortSec=
 #DefaultRestartSec=100ms
 #DefaultStartLimitIntervalSec=10s
